### PR TITLE
Socialisation Training tweaks

### PR DESCRIPTION
**Pull request by Charlotte-Morgan**
@adissadis - please take a look, minor changes to your files to update links and fix some wording.
Would be great to get this merged for 3.2.1 bug fix release this week.

### CODE REVIEW

**Comment by Charlotte-Morgan** [comment]
@timlinux - I'd like to get this in 3.2.1 please

File: docs/source/training/socialisation/inasafe_concepts.rst [hunk 1]
```diff
@@ -1,33 +1,58 @@
 .. _inasafe_concepts:
 
-Key concepts in disaster management planning with |project_name|
-================================================================
-In this section we explain the basic concepts of |project_name| and the merits
-of disaster management planning.
+ |project_name| Concepts
```
Comment (adissadis) [inline docs/source/training/socialisation/inasafe_concepts.rst]: Hi @Charlotte-Morgan this become warning when i tried to build your file.

It said:
/inasafe-doc/docs/source/training/socialisation/inasafe_concepts.rst:8: WARNING: Block quote ends without a blank line; unexpected unindent

**Comment by timlinux** [comment]
Hi @Charlotte-Morgan 

> @timlinux - I'd like to get this in 3.2.1 please

Ok noted I will make sure we publish it at the same time we release 3.2.1

**Comment by Charlotte-Morgan** [comment]
Hi @rduivenvoorde I'm merging this now but we expect to have additional files to add to the socialisation training before the release. eg #2354 
@adissadis @harrymahar are working on new files that should be ready on Friday :)

**Comment by rduivenvoorde** [comment]
I build develop here locally, no build errors, nice clean build :-)

**Pull request merged**